lib/services/dbus.c

```diff
@@ -507,6 +523,8 @@ pcmk_dbus_watch_dispatch(gpointer userdata)
         crm_err("DBus encountered OOM while attempting to dispatch %p (%s)",
                 client, dbus_watch_flags_to_string(flags));
     }
+
```

Comment:
maybe we should have an "else" here -- if we don't have enough memory to do dbus_watch_handle(), we probably won't have enough to dispatch either, and might as well wait for the next time. although, it's less than ideal already, that if we get oom, we wait until there's more data on the fd before trying again. of course, if we get oom, we'll probably segfault or fast-exit before long anyway, so maybe it's not worth worrying about